server/app/services/question/types/QuestionDefinitionConfig.java

```diff
@@ -53,20 +53,12 @@ public interface RequiredQuestionText {
   }
 
   public interface RequiredQuestionHelpText {
-    RequiredValidationPredicates setQuestionHelpText(LocalizedStrings questionHelpText);
-  }
-
-  public interface RequiredValidationPredicates {
-    Builder setValidationPredicates(QuestionDefinition.ValidationPredicates validationPredicates);
+    Builder setQuestionHelpText(LocalizedStrings questionHelpText);
   }
```

Comment:
Removing `RequiredValidationPredicates` makes `validationPredicates()` not required to be set at compile-time.

Reply:
QQ- then why was it implemented before? Can you please explain